### DISCUSSION TITLE
为什么alexnet只训练四层和五层，不训练前面几层

### DISCUSSION THREAD

**Discussion by aboy2018** [STVIR/pysot#354] [General] [answered]
您好，我想问一下，在加载预训练模型时，alexnet只对4，5层训练，为什么不对前面几层训练，是有什么考虑吗；
这是不是意味着开源的最好模型和原始alexnet前面几层参数一样，如果我加载的预训练模型是开源的最好模型，是不是也只需要对4，5层训练，还望大神指教，谢谢。

**Comment by ZhiyuanChen** [comment, marked as answer]
可以，但没必要。
